### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ ExportBacpacSettings_Screenreader: Name is missing for the table “Export Bacpac Settings”

### ISSUE THREAD

**Issue by mspatil110** [microsoft/azuredatastudio#6948]
**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MAC iOS version: 10.16.4

**Additional Details:**
Tools Name: Accessibility insight tool
MAS References: MAS1.3.1, MAS4.1.2.

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select forth radio button.
9. Navigate till next button & hit enter.
10. Launch Voiceover and verify that name is present or not for table by navigating on table.

**Actual:**
Name is missing for the table “Export Bacpac Settings”

**Expected:**
Name should be  “Export Bacpac Settings”  for “Export Bacpac Settings”table.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will face difficulty to understand the table.

**MAS Reference:**

MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

MAS4.1.2-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%204.1.2%20%E2%80%93%20Name,%20Role,%20Value.docx?d=w248054a65e6847719e1e242fb5025730&csf=1&e=v1KF77
